modules/documents/client/directives/documents.manager.directive.js

```diff
@@ -228,7 +229,15 @@ angular.module('documents')
 					self.selectNode(doc.model.id);
 				};
 
-				self.selectNode = function (nodeId) {
+				self.gotoDoc = function(docID) {
+					Document.lookup(docID) //docID is an objectID
```

Comment:
localhost:3000/docs?file=andObjectIDTheyDon'tHaveAccessTo

Use Case: They used to have access, but now don't.  Are we going to display a message?